.github/workflows/site.yml

```
@@ -0,0 +1,80 @@
+  run:
+    shell: bash
+
+jobs:
+  # Build job
+  build:
+    runs-on: ubuntu-latest
+    env:
+      HUGO_VERSION: 0.126.0
+    steps:
+      - name: Install Hugo CLI
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:89: Double quote to prevent globbing and word splitting

(shellcheck)